build.gradle.kts

```diff
@@ -126,8 +120,9 @@ dependencies {
     implementation ("no.nav.helse:syfosm-common-ws:$smCommonVersion")
     implementation ("no.nav.helse:syfosm-common-kafka:$smCommonVersion")
     implementation ("no.nav.helse:syfosm-common-diagnosis-codes:$smCommonVersion")
-    implementation ("no.nav.helse:syfosm-common-mq:$smCommonVersion")
-
+    implementation ("no.nav.helse:syfosm-common-mq:$smCommonVersion") {
```

Comment:
Har denne appen mq? Og webservices (soap)? Kanskje vi kan fjerne syfosm-common-ws og syfosm-common-mq?

Reply:
Jeg tror kanskje denne kan fjernes: dok-journalfoering-hendelse-v1

Og hvis appen ikke bruker ws så kan nok alt av cxf og jaxws fjernes også :)

Reply:
Den excluden var forresten kun nødvendig i syfosminfotrygd, og vil være nødvendig i syfoservice-mq-producer, siden de to går mot gammel kømanager. Alle andre er migrert og kan bruke nyeste klienten 😊

Reply:
> Jeg tror kanskje denne kan fjernes: dok-journalfoering-hendelse-v1

Jeg tror dette brukes i BehandlingService https://github.com/navikt/syfosmpapirmottak/blob/master/src/main/kotlin/no/nav/syfo/service/BehandlingService.kt#L30

Reply:
Det er kanskje avroskjemaet for joark-topicen? I så fall er den i bruk ja :D

Reply:
Jepp!